Exceptions/PermExceptions.cs/UserNotFound.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+[Serializable]
+public class UserNotFound : Exception
```

Comment:
When declaring Exception class, make sure to include the "Exception" suffix, so in this case it would be `UserNotFoundException`